src/web/components/ArticleContainer.tsx

```diff
@@ -53,8 +53,8 @@ const articleAdStyles = css`
             margin-right: -328px;
         }
 
-        ${from.wide}  {
-            margin-right: -388px;
+        ${from.wide} {
+            margin-right: -408px;
```

Comment:
Oh, you meant the value has changed. So yeah, you're right, on master I have

```
        ${from.wide}  {
            margin-right: -388px;
        }
```
So -408 is wrong

Reply:
I put it back

Reply:
👍